### PR TITLE
Compiler doc update

### PR DESCRIPTION
**Pull request by johnspackman**

### CODE REVIEW

**Comment by johnspackman** [comment]
closing in favour of committing into to hkollmann_doc_v7

File: docs/core/classes.md [hunk 1]
```diff
@@ -345,22 +308,20 @@ qx.Class.define("qx.test.Cat", {
...
 The syntax for accessing static variables simply is
-`this.constructor.someStaticVar`. Please note, for `this.constructor` to be
-available, the class must be a derived class of `qx.core.Object` , which is
-usually the case for regular, non-static classes.
-
-Instead of `this.constructor` you can also use the alternative syntax
-`this.self(arguments)`.
+`qx.test.Animal.someStaticVar`.  Please note that `this.constructor` is not
+recommended because it always refers to the subclass of the object, and there
+can change change the implementation at run time.  Always specify class names
```
Comment (cboulanger) [inline docs/core/classes.md:320]: "and there can change change" - probably something wrong here?

File: docs/development/README.md [hunk 2]
```diff
@@ -8,13 +8,10 @@ application.
...
 > from the Python-based `generator.py` script to a NodeJS-based `qx` CLI. The
-> python tool chain will still work for all 6.\* releases. However, it has been
-> marked deprectated and will be removed in version 7.0.0. Since we won't be
-> able to maintain the old toolchain or provide support for it, we strongly
-> recommend to use the new one for your projects. To convert your existing
-> applications, please review
-> [our guide to migrate your applications to the new NodeJS-based tooling](compiler/migration.md)
-> .
+> python tool chain will still work for all 6.\* releases but was
+> marked deprectated has been removed in version 7.0.0.  Because the v7 release is
```
Comment (cboulanger) [inline docs/development/README.md:12]: some syntax problem here?

**Review by cboulanger — APPROVED** [review]

File: docs/core/classes.md (hunk 1 above)
Comment (johnspackman) [reply docs/core/classes.md:320]: done in hkollmann_doc_v7

File: docs/development/README.md (hunk 2 above)
Comment (johnspackman) [reply docs/development/README.md:12]: also done

**Pull request closed without merging**